SPECS/newt/newt.spec

```diff
@@ -30,13 +31,26 @@ Requires:	%{name} = %{version}
 %description	devel
 It contains the libraries and header files to create applications
 
+%package -n python3-newt
+%{?python_provide:%python_provide python3-newt}
```

Comment:
What's the point of this macro?

Reply:
I think it is `Provides: python3-newt`, i've removed it as we already have a package for it.